keycloak_oidc/drf/__init__.py

```diff
@@ -0,0 +1,4 @@
+# from django.conf import settings
+#
+# assert hasattr(settings, 'OIDC_RP_CLIENT_ID'), "OIDC_RP_CLIENT_ID must be set in settings"
+# assert hasattr(settings, 'OIDC_RP_CLIENT_SECRET'), "OIDC_RP_CLIENT_SECRET must be set in settings"
```

Comment:
why add this commented out?

Reply:
Oops! Forgot to remove. Fixed in later commit.